index.js

```diff
@@ -2,10 +2,16 @@
 
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 1", "color: red") 
+// let counter = 1;
+
+// if (counter === 1) {
+//   let counter = 2;
+//   console.log(counter)
+// }
```

Comment:
Using `let` is the right idea here, but there's a bit of extra work you're doing - there's no need to have the conditional statement and create another variable:

```js
// declare the variable and assign a value of 1
let counter = 1
// reassign it to a new value
counter = 2
```